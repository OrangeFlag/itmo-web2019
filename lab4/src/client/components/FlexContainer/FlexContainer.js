import styled from "styled-components";

const FlexContainer = styled("div")`
  display: flex;
  flex-flow: row wrap;
  align-items: left;

  // width: 100%;


  * {
    margin-right: 16px;
  }
`;

export default FlexContainer