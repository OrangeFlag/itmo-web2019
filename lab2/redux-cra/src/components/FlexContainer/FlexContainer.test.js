import React from 'react';
import {shallow} from 'enzyme';
import FlexContainer from './FlexContainer';

describe('Container tests', () => {
    it('should render correctly', () => {
        const container = shallow(<FlexContainer>+</FlexContainer>);

        expect(container).toMatchSnapshot();
    });
});