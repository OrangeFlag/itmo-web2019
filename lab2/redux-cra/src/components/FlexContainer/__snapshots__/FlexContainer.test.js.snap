// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container tests should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: flex;
  flex-flow: row wrap;
  align-items: left;

  // width: 100%;


  * {
    margin-right: 16px;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  +
</StyledComponent>
`;
